### PR TITLE
[PM-5956] Delete Unused State

### PR DESCRIPTION
**Pull request by justindbaur**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [x] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Little bit of housekeeping of state that we've either already migrated to state providers already, don't have any reference to it elsewhere, or seems pretty pointless (installedVersion). 

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

File: libs/common/src/platform/models/domain/account.ts [hunk 1]
```diff
@@ -66,13 +66,6 @@ export class DataEncryptionPair<TEncrypted, TDecrypted> {
   decrypted?: TDecrypted[];
 }
 
-// This is a temporary structure to handle migrated `DataEncryptionPair` to
-//  avoid needing a data migration at this stage. It should be replaced with
-//  proper data migrations when `DataEncryptionPair` is deprecated.
-export class TemporaryDataEncryption<TEncrypted> {
-  encrypted?: { [id: string]: TEncrypted };
-}
-
```
Comment (MGibson1) [inline libs/common/src/platform/models/domain/account.ts:75]: 🥳

File: libs/common/src/platform/models/domain/global-state.ts [hunk 2]
```diff
@@ -1,16 +1,9 @@
-import { ThemeType } from "../../enums";
-
 export class GlobalState {
-  installedVersion?: string;
```
Comment (MGibson1) [inline libs/common/src/platform/models/domain/global-state.ts:4]: Thoughts on deleting this with a migrator?

**Review by MGibson1 — COMMENTED** [review]
🙇 🙇 🙇 thank you for doing this housekeeping!

File: libs/common/src/platform/models/domain/global-state.ts (hunk 2 above)
Comment (justindbaur) [reply libs/common/src/platform/models/domain/global-state.ts:4]: Good thinking, did it.

**Review by MGibson1 — DISMISSED** [review]

**Review by MGibson1 — APPROVED** [review]

**Pull request merged**